public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -287,38 +278,15 @@ export function GrafanaEvaluationBehaviorStep({
           )}
         </Stack>
 
-        {folder?.uid && isEditingGroup && (
-          <EditRuleGroupModal
-            ruleGroupIdentifier={{
-              dataSourceName: GRAFANA_RULES_SOURCE_NAME,
-              groupName: existingGroup?.name ?? '',
-              namespaceName: folder?.uid ?? '',
-            }}
-            rulerConfig={GRAFANA_RULER_CONFIG}
-            onClose={() => closeEditGroupModal()}
-            intervalEditOnly
-            hideFolder={true}
-          />
-        )}
         {folder?.title && group && (
           <div className={styles.evaluationContainer}>
             <Stack direction="column" gap={0}>
               <div className={styles.marginTop}>
                 <Stack direction="column" gap={1}>
                   {getValues('group') && getValues('evaluateEvery') && (
```

Comment:
not related to this PR but, we can use the const `evaluateEvery` instead of `getValues('evaluateEvery')`, and the const `group` instead of `getValues('group')`, right? or there is a reason for it?

Reply:
I don't quite remember but I'm somewhat conservative when it comes to these things – I don't recall why it's inline here :D